ci-operator/step-registry/gcp/provision/private-dns-zone-in-service-project/gcp-provision-private-dns-zone-in-service-project-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+echo "Reading variables from ${SHARED_DIR}/customer_vpc_subnets.yaml..."
+NETWORK=$(yq-go r "${SHARED_DIR}/customer_vpc_subnets.yaml" 'platform.gcp.network')
+
+BASE_DOMAIN="$(< ${CLUSTER_PROFILE_DIR}/public_hosted_zone)"
+CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  GCP_PROJECT="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GCP_PROJECT}"
+fi
+
+gcloud dns managed-zones create "${CLUSTER_NAME}-private-zone" --description "Pre-created private DNS zone" --visibility "private" --dns-name "${CLUSTER_NAME}.${BASE_DOMAIN%.}." --networks "${NETWORK}"
```

Comment:
should be `"${CLUSTER_NAME}.${BASE_DOMAIN}.`,right?

Reply:
No, I want to remove the ending dot if any.